package.json

```diff
@@ -5,49 +5,43 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
+    "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
     "build-themes": "devextreme build",
     "postinstall": "npm run build-themes"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~11.1.0",
-    "@angular/cdk": "^8.0.0",
-    "@angular/common": "~11.1.0",
-    "@angular/compiler": "~11.1.0",
-    "@angular/core": "~11.1.0",
-    "@angular/forms": "~11.1.0",
-    "@angular/platform-browser": "~11.1.0",
-    "@angular/platform-browser-dynamic": "~11.1.0",
-    "@angular/router": "~11.1.0",
-    "devextreme": "^20.2.4",
-    "devextreme-angular": "^20.2.4",
-    "devextreme-schematics": "^1.2.10",
+    "@angular/animations": "~12.0.2",
+    "@angular/cdk": "^11.2.2",
+    "@angular/common": "~12.0.2",
+    "@angular/compiler": "~12.0.2",
+    "@angular/core": "~12.0.2",
+    "@angular/forms": "~12.0.2",
+    "@angular/platform-browser": "~12.0.2",
+    "@angular/platform-browser-dynamic": "~12.0.2",
+    "@angular/router": "~12.0.2",
+    "devextreme": "^21.1.3",
+    "devextreme-angular": "^21.1.3",
+    "devextreme-schematics": "^1.2.14",
     "rxjs": "~6.6.0",
-    "tslib": "^2.0.0",
-    "zone.js": "~0.11.3"
+    "tslib": "^2.1.0",
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1101.1",
-    "@angular/cli": "~11.1.1",
-    "@angular/compiler-cli": "~11.1.0",
+    "@angular-devkit/build-angular": "~12.0.2",
+    "@angular/cli": "~12.0.2",
+    "@angular/compiler-cli": "~12.0.2",
```

Comment:
I thought we have a problem with angular 12

Reply:
We have turned off strict mode in our cli
https://github.com/DevExpress/devextreme-cli/pull/301
And we will return it after fix